app/db.py

```diff
@@ -1,16 +1,230 @@
+            self.id,
+            self.tweet_id,
+            self.form,
+            self.root_tweet_city,
+            self.root_tweet_state,
+            self.root_tweet_lat,
+            self.root_tweet_long,
+            self.root_tweet_date,
+            self.root_tweet_force_rank,
+            self.sent_tweet_id,
+            self.received_tweet_id,
+            self.in_reply_to_id,
+            self.tweeter_id,
+            self.conversation_status,
+            self.tweet_text,
+            self.checks_made,
+            self.reachout_template,
+            self.isChecked
+        )
+
+
+class Training(Base):
+
+	__tablename__ = "training"
+
+	id = Column(Integer, primary_key=True)
+	tweets = Column(String)
+	labels = Column(Integer)
+
+	def __repr__(self):
+		return(
+			"id:{}, tweets:{}, labels:{}").format(
+			self.id,
+			self.tweets,
+			self.labels
+			)
+
+
+class BotScripts(Base):
+
+    __tablename__ = "bot_scripts"
+
+    script_id = Column(Integer, primary_key=True, nullable=False, unique=True)
+    script = Column(String(255))
+    convo_node = Column(Integer)
+    use_count = Column(Integer)
+    positive_count = Column(Integer)
+    success_rate = Column(Float)
+    active = Column(Boolean)
+
+    def __repr__(self):
+        return (
+            "script_id:{}, script:{}, convo_node:{}, use_count:{}, positive_count:{}, success_rate{}, active:{}").format(
+            self.script_id,
+            self.script,
+            self.convo_node,
+            self.use_count,
+            self.positive_count,
+            self.success_rate,
+            self.active
+            )
+
+
+class ScriptTesting(Base):
+
+	__tablename__ = "script_testing"
+
+	incident_id = Column(Integer, primary_key=True, nullable=False, unique=True)
+	script_path = Column(String(100))
+	success = Column(Boolean)
+
+	
+	def __repr__(self):
+		return (
+			"incident_id:{}, script_path:{}, success:{}").format(
+			self.incident_id,
+			self.script_path,
+			self.success
+			)
+
+
+class Sources(Base):
+
+	__tablename__ = "sources"
+
+	source_id = Column(Integer, primary_key=True, nullable=False, unique=True)
+	incident_id = Column(Integer, ForeignKey("force_ranks.incident_id"))
+	source = Column(String(255))
+
+	
+	def __repr__(self):
+		return (
+			"source_id:{}, incident_id:{}, sources:{}").format(
+			self.source_id,
+			self.incident_id,
+			self.source
+			)
+
+
+class Tags(Base):
 
-from app.models import ForceRanks, Conversations, BotScripts, ScriptTesting, Tags, Sources
+	__tablename__ = "tags"
 
-db_url = os.getenv('DB_URL')
+	tags_id = Column(Integer, primary_key=True, nullable=False, unique=True)
+	incident_id = Column(Integer, ForeignKey("force_ranks.incident_id"))
+	tag = Column(String(40))
 
+	
+	def __repr__(self):
+		return (
+			"tags_id:{}, incident_id:{}, sources:{}").format(
+			self.tags_id,
+			self.incident_id,
+			self.tag
+			)
 
 class Database(object):
 
```

Comment:
This is a good start to the circular dependency issue that was being caused when trying to refactor the bot.py to use dynamic dispatching. Combining the db.py and model.py file into db.py was the easiest solution and made the most sense.